.gitignore

```diff
@@ -3,3 +3,4 @@
 *.html
 *.css
 *.js
+tags
```

Comment:
It's the tagfile for vim users, which is kind-of the standard filename for generating jump-to-definitions. I'll remove this since it probably shouldn't be in the repo.